ystia/forge
# Bugfix/gh 91 cleanup bootstrap hp

# Pull Request description

## Description of the change

Fixed cleanup issues where :
- AnsibleRuntime component delete interface attempts to uninstall its prerequisites, it has the side effect of removing packages installed by default on the system
- ConsulServer component doesn't have a delete interface, so doesn't cleanup /var/consul directory, so that next install using the same directoy wil retrieve consul data from the previous install

### What I did

`org/ystia/ansible/linux/ansible/types.yaml`:
Removing the delete interface whose playbook was uninstalling prerequisite packages

`org/ystia/experimental/consul/linux/ansible/playbooks/consul_cleanup.yaml`:
New playbook removing consul data directory to ensure a new install on the same setup won't retrieve data from a previous install

`org/ystia/experimental/consul/linux/ansible/types.yaml`:
Added a delete interface to cleanup consul data directory

### How to verify it

See How to verify section of Yorc boostrap pull request https://github.com/ystia/yorc/pull/408

### Description for the changelog

Bad cleanup for components used in yorc bootstrap ([GH-91](https://github.com/ystia/forge/issues/91))

## Applicable Issues

Fixes #91
